contrib/globalsign/mgo/mgo.go

```diff
@@ -0,0 +1,395 @@
+package mgo
```

Comment:
We should add a package level comment:

```go
// Package mgo ...
```

And a canonical import path:

```go
package mgo // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/globalsign/mgo"
```

See other integrations for examples

Reply:
Yup, will do.  Thanks.

Reply:
Done